language/bytecode-verifier/bytecode-verifier-tests/src/unit_tests/multi_pass_tests.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use bytecode_verifier::{
+    constants, instantiation_loops::InstantiationLoopChecker, DuplicationChecker,
+    InstructionConsistency, RecursiveStructDefChecker, ResourceTransitiveChecker, SignatureChecker,
+};
+use proptest::prelude::*;
+use vm::CompiledModule;
+
+proptest! {
+    #[test]
+    fn check_verifier_passes(module in CompiledModule::valid_strategy(20)) {
```

Comment:
this increases code coverage of the verifier tests and it's a good way to check all "syntactic" checks given the module generated can contains almost all acceptable constructs

Reply:
we cannot and would not pass any code unit checks, and generally any test that wants a cfg. This module would never verify logically. But instructions are correct on their own. Does it makes sense?
Those proptests only give you syntactic checks, so to speak.
The idea behind the name is that is testing some checks not all, but we may find a better name. verify_module seems too overloaded to me